js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeExtensionsEnabled.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package ai.onnxruntime.reactnative;
+
+import ai.onnxruntime.OrtException;
+import ai.onnxruntime.OrtSession.SessionOptions;
+import ai.onnxruntime.extensions.OrtxPackage;
+
+class OnnxruntimeExtensions {
```

Comment:
not specifying `public` here for this class as other wise compiler would complain about the class name not match the file name. but in our case, it should be fine as if not specified, will by default package-specific. We only will call and use this class within the same package bundle `ai.onnxruntime.reactnative`